docs/articles_en/openvino-workflow/model-optimization.rst

```diff
@@ -13,57 +10,103 @@ Model Optimization Guide
-   the OpenVINO Intermediate Representation format (IR) right away.
+To learn about the full scope of the framework, its installation, and technical details, visit
+both `the NNCF repository <https://github.com/openvinotoolkit/nncf?tab=readme-ov-file>`__ and
+`NNCF API documentation <https://openvinotoolkit.github.io/nncf/autoapi/nncf/>`__.
 
-Post-training Quantization is the fastest way to optimize an arbitrary DL model and should
-be applied first, but it is limited in terms of achievable accuracy-performance trade-off.
-The recommended approach to obtain OpenVINO quantized model is to convert
-`a model from original framework <https://huggingface.co/models>`__ to ``ov.Model``
-and ensure that the model works correctly in OpenVINO, for example, by calculating the
-model metrics. Then, ``ov.Model`` can be used as input for the ``nncf.quantize()``
-method to get the quantized model or as input for the ``nncf.compress_weights()``
-method to compress weights of Large Language Models (see the diagram below).
 
-In case of unsatisfactory accuracy or performance after Post-training Quantization,
-Training-time Optimization can be used as an option.
 
 .. image:: ../assets/images/DEVELOPMENT_FLOW_V3_crunch.svg
 
-Once the model is optimized using the aforementioned methods, it can be used for
-inference using the regular OpenVINO inference workflow. No changes to the inference
-code are required.
 
-.. image:: ../assets/images/WHAT_TO_USE.svg
+.. tab-set::
 
-Additional Resources
-####################
+   .. tab-item:: Installation
+      :sync: install
+
+      .. tab-set::
+
+         .. tab-item:: PyPI
+            :sync: pip
+
+            .. code-block::
+
+               pip install nncf
+
+         .. tab-item:: Conda
+            :sync: conda
 
-- :doc:`Post-training Quantization <model-optimization-guide/quantizing-models-post-training>`
-- :doc:`Training-time Optimization <model-optimization-guide/compressing-models-during-training>`
-- :doc:`Weight Compression <model-optimization-guide/weight-compression>`
-- :doc:`Deployment optimization <running-inference/optimize-inference>`
-- `Hugging Face Optimum Intel <https://huggingface.co/docs/optimum/intel/optimization_ov>`__
+            .. code-block::
+
+               conda install -c conda-forge nncf
+
+      For more installation details, see the page on
+      `NNCF Installation <https://github.com/openvinotoolkit/nncf/blob/develop/docs/Installation.md>`__.
+
+   .. tab-item:: System Requirements
+      :sync: sys-req
+
+      | Full requirement listing is available in:
+      | `NNCF GitHub Repository <https://github.com/openvinotoolkit/nncf?tab=readme-ov-file#system-requirements>`__
+      |
+      | Note that to optimize a model, you will need to install this model's framework as well.
+      | Install NNCF in the same Python environment as the framework. For a list of recommended
+      | framework versions, see the
```

Comment:
There is some issue with the text going on multiple lines.

Reply:
Fixed in https://github.com/openvinotoolkit/openvino/pull/25981/commits/dd98ac3b09867d508b97a500c73be077c7d361cd